lib/src/controls/better_player_controls_state.dart

```diff
@@ -290,23 +290,33 @@ abstract class BetterPlayerControlsState<T extends StatefulWidget>
   ///Track selection is used for HLS / DASH videos
   ///Resolution selection is used for normal videos
   void _showQualitiesSelectionWidget() {
-    // HLS / DASH
+   // HLS / DASH
     final List<String> asmsTrackNames =
         betterPlayerController!.betterPlayerDataSource!.asmsTrackNames ?? [];
-    final List<BetterPlayerAsmsTrack> asmsTracks =
+    final List<BetterPlayerAsmsTrack> asmstracks =
         betterPlayerController!.betterPlayerAsmsTracks;
-    final List<Widget> children = [];
-    for (var index = 0; index < asmsTracks.length; index++) {
-      final track = asmsTracks[index];
+    final List<Widget> children = []; 
+  
+   final sortedTracks = [
+      ...(asmstracks
```

Comment:
Why not just
...asmstracks..sort((a, b) => (b.height ?? 0).compareTo(a.height ?? 0))
?